cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -378,13 +423,15 @@ func (oi *OVFImporter) Import() error {
 	if err := oi.paramValidator.ValidateAndPopulate(oi.params); err != nil {
 		return err
 	}
-	if err := oi.importDiskFilesToImages(); err != nil {
-		oi.imageDeleter.DeleteImagesIfExist(oi.images)
+	if err := oi.importDisksVMDKFiles(); err != nil {
+		oi.resourceDeleter.DeleteImagesIfExist(oi.images)
```

Comment:
Could you clarify why do we need to delete images? I thought that they wouldn't be created at all. Do I miss anything?

Reply:
good find -- I think this `DeleteImagesIfExist` can be removed here since mahmoud is only making disks at this point.

Reply:
I am not very familiar with this part of code, therefore it would be good to know for the future, why do we need to create an image for a bootable disk at all to import an instance. May I ask you to clarify it, please?

Reply:
Because we need to import the boot disk using image importer to be passed on al the stages of image import, also to be translated, and this is demonstrated in the design document, am I answered your question?

Reply:
I ask about a slightly different thing. I see that you use this image here: https://github.com/GoogleCloudPlatform/compute-image-tools/blob/1675a8ccca681f85284255749386ec6e8d786567/cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go#L389

However, do we really need an image in the end to create an instance? Can we modify the workflow in the way that it doesn't require an image but only a reference to a bootable disk?

@EricEdens, do we have any technical limitations here?

Reply:
We create disks for data disks only, and create image for boot disk, and use these data disks to translate the boot-disk image, then the boot disk will be created from this image, it will be created on the final instance, and the created data disks will be attached to the final instance.